pages.es/linux/pw-metadata.md

```diff
@@ -0,0 +1,33 @@
+# pw-metadata
+
+> Supervisa, establece y elimina metadatos en objetos PipeWire.
+> Vea también: `pipewire`, `pw-mon`, `pw-cli`.
+> Más información: <https://docs.pipewire.org/page_man_pw-metadata_1.html>.
+
+- Muestra metadatos en formato `predeterminado`:
+
+`pw-metadata`
+
+- Muestra metadatos con ID 0 en `configuración`:
+
+`pw-metadata {{-n|--name}} {{configuración}} {{0}}`
```

Comment:
```suggestion
`pw-metadata {{-n|--name}} {{settings}} {{0}}`
```
I don't think this should be translated as `settings` really exist on systems with pipewire so it's a ready-to-use example.